api/cost/cost-request-fn.ts

```diff
@@ -1,15 +1,17 @@
+import { injectLambdaContext } from "@aws-lambda-powertools/logger";
+import { captureLambdaHandler } from "@aws-lambda-powertools/tracer";
+import { SendMessageCommand } from "@aws-sdk/client-sqs";
 import middy from "@middy/core";
-import { sqsClient } from "../../utils/aws-sdk/sqs";
+import errorLogger from "@middy/error-logger";
+import inputOutputLogger from "@middy/input-output-logger";
 import { SQSEvent, SQSRecord } from "aws-lambda";
-import { logger } from "../../utils/logging";
+import JSONErrorHandlerMiddleware from "middy-middleware-json-error-handler";
```

Comment:
Did you reorder the imports, and if so is there a way you normally organize?

Reply:
I did reorder the imports. I used the default ordering from the TypeScript Language Server, `tsserver`, (so what would happen if you used the VS Code command)